packages/compiler/core/cli/args.ts

```diff
@@ -33,8 +34,9 @@ export async function getCompilerOptions(
 ): Promise<[CompilerOptions | undefined, readonly Diagnostic[]]> {
   const diagnostics = createDiagnosticCollector();
   const pathArg = args["output-dir"] ?? args["output-path"];
+  const tspconfigDir = args["tspconfig-dir"] ?? cwd;
```

Comment:
tspConfigDir

Reply:
I think it makes the most sense to leave the default being a folder because it will then search the hierarchy for the default filename.